app/overlays/tgl-h/ipc4_overlay.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32=y
 CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
 CONFIG_COMP_VOLUME_WINDOWS_FADE=y
 CONFIG_COMP_DRC=y
+CONFIG_COMP_ARIA=y
```

Comment:
is this for chrome and linux only？ if there is no real usage for aria in chrome and linux, keep as N is reasonable.

Reply:
I would like to enable it since we have already add support in .tmol file, and we will need this component in the next generation device

Reply:
commit message is for tgl device, and tgl device does not need aria, right?
which device need which feature is pre-defined, if mtl need aria, we can enable it in mtl config, not for tgl.

I guess enable it in tgl just for convenient test purpose, maybe @kv2019i @lgirdwood  can have some input.

no harmful, just add more code to tgl fw.

Reply:
okay, remove from this PR, we can enable it later in another PR.